routes/stats.py

```diff
@@ -92,6 +101,10 @@ def chart_stats(session=None, num_days=7):
             percent = round((parsed / sent) * 100, 2) if sent else 0
             parsed_data.append(percent)
 
+            # Get query count for day
```

Comment:
A bit ugly, since this executes a separate SQL query for each day; but maybe OK for this non-critical use case.